federation-2/router-bridge/Cargo.toml

```diff
@@ -45,3 +45,11 @@ tracing-test = "0.2.1"
 [build-dependencies]
 deno_core = "0.142.0"
```

Comment:
should deno_core be marked as optional?

Reply:
No because `cargo doc` runs the equivalent of `cargo check`. It’s only the C++ bits of V8 that are disabled apparently, all the Rust bits of Deno and rusty_v8 and still there.